confluent_kafka/avro/__init__.py

```diff
@@ -53,8 +53,7 @@ def __str__(self):
 
 
 from confluent_kafka.avro.cached_schema_registry_client import CachedSchemaRegistryClient
-from confluent_kafka.avro.serializer import (SerializerError,
-                                             KeySerializerError,
+from confluent_kafka.avro.serializer import (KeySerializerError,
```

Comment:
I think we want the `SerializerError` import so users only have to `from confluent_kafka.avro import (X, Y, Z)` and don't have to dig all the way into `confluent_kafka.avro.serializer`.